core/src/main/java/nu/yona/server/analysis/service/ActivityService.java

```diff
@@ -169,6 +169,12 @@ private void createAndSaveInactivityDays(UserAnonymizedDto userAnonymized, Set<G
 		weekActivity.createRequiredInactivityDays(userAnonymized, goals, LevelOfDetail.WEEK_OVERVIEW, missingInactivities);
 	}
 
+	private long getTotalPageableItems(Set<BuddyDto> buddies, ChronoUnit timeUnit)
```

Comment:
Mmm. It's just an overload of the other method. One takes a ``userAnonymized`` and returns the number of pageable items for it, the other takes a set of buddies and does the same. I like to keep these method names the same.